test/Algebra/Graph/Test/API.hs

```diff
@@ -292,6 +293,51 @@ graphAPI = API
     , compose                    = G.compose
     , box                        = G.box }
 
+-- | The API of 'UG.Graph.
```

Comment:
Please check all generated docs: the closing quote is missing here.

Reply:
Thanks! I've checked all the generated docs and fixed some small issues like this, I hope it's all good now!